sundog/view/documents.py

```diff
@@ -0,0 +1,329 @@
+from datatableview import Datatable
+from datatableview.columns import CompoundColumn, DisplayColumn, TextColumn
+from datatableview.helpers import make_processor, through_filter
+from datatableview.views import XEditableDatatableView
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.db.models import (
+    DateTimeField,
+    ForeignKey,
+    Model,
+    SET_NULL,
+)
+from django.forms.models import ModelForm
+from django.forms.widgets import Select, SelectMultiple
+from django.template.defaultfilters import date
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.views.generic.edit import UpdateView
+from fm.views import AjaxCreateView, AjaxDeleteView, AjaxUpdateView
+from multiselectfield import MultiSelectField
+from settings import SHORT_DATETIME_FORMAT
+from sundog.models import NONE_CHOICE_LABEL
+from sundog.routing import decorate_view, route
+from sundog.utils import LongCharField
+from tinymce.models import HTMLField
+
+
+class Document(Model):
```

Comment:
I definitely do not like to have all together in the same file, it just destroys the separate concerns structure which Django has so option 2 is better IMO. Thou i don't think we can afford to refactor previous code but to start doing stuff this way with new structure and maybe try and refactor by pieces (the way that is agile development is supposed to happen).